ibex_icache.core

```diff
@@ -0,0 +1,19 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:ip:ibex_icache:0.1"
+description: "IBEX_ICACHE DV sim target"
+filesets:
+  files_rtl:
+    files:
+      - shared/rtl/ram_1p.sv
+      - rtl/ibex_icache.sv
+    file_type: systemVerilogSource
+
+targets:
+  default: &default_target
+    filesets:
+      - files_rtl
+    toplevel: ibex_icache
+    default_tool: vcs
```

Comment:
Why is this file necessary? The icache is already included in the `ibex_core.core` file.

Reply:
I wanted to create a core file just for the `ibex_icache.sv`design since the `ibex_core.core` file includes all of the other RTL code, which isn't needed for the icache testbench.

Reply:
In this case, please remove the cache-related files from the main core file, and depend on this file instead.

Reply:
Sure, will do